src/Views/Game.hs

```diff
@@ -21,6 +23,7 @@ createGameDOM api gapi = do
 
   createGameChangeNameDOM api
   createUpdateMaxNumberPlayersDOM api gapi
+  createSetPasswordDOM api
```

Comment:
When a client isn't the creator of the game we don't even need to show the DOM object for setting the password.
The same goes for any kind of configuration which can be done for a game.